### PR TITLE
fix(swap): format market price

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-910](https://emurgo.atlassian.net/browse/YOMO-910)

[YOMO-910]: https://emurgo.atlassian.net/browse/YOMO-910?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jorbuedo — APPROVED** [review]

**Comment by jorbuedo** [comment]
@banklesss I think we added yesterday the priceDenomination to the state, so you don't have to calculate it on the app.

File: apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/LimitPriceWarning/LimitPriceWarning.tsx [hunk 1]
```diff
@@ -19,14 +20,17 @@ export const LimitPriceWarning = ({onSubmit, orderData}: LimitPriceWarningProps)
   const {numberLocale} = useLanguage()
   const strings = useStrings()
   const limitPrice = new BigNumber(orderData.limitPrice ?? 0).toFormat(numberLocale)
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/LimitPriceWarning/LimitPriceWarning.tsx]: limit also needs to be adjusted in the orderData is raw price, not denominated to UI

Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/LimitPriceWarning/LimitPriceWarning.tsx]:
```suggestion
  const limitPrice = Quantities.format(
     orderData.limitPrice ?? Quantities.zero,
     orderData.tokens.priceDenomination,
   )
```

File: apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/LimitPriceWarning/LimitPriceWarning.tsx [hunk 2]
```diff
@@ -19,14 +20,17 @@ export const LimitPriceWarning = ({onSubmit, orderData}: LimitPriceWarningProps)
...
   const {closeModal} = useModal()
-
   const tokenToSellInfo = useTokenInfo({wallet, tokenId: orderData.amounts.sell.tokenId})
   const tokenToSellName = tokenToSellInfo.ticker ?? tokenToSellInfo.name ?? '-'
   const tokenToBuyInfo = useTokenInfo({wallet, tokenId: orderData.amounts.buy.tokenId})
   const tokenToBuyName = tokenToBuyInfo.ticker ?? tokenToBuyInfo.name ?? '-'
+  const marketPrice = Quantities.format(
+    orderData.selectedPoolCalculation?.prices.market ?? Quantities.zero,
+    orderData.tokens.priceDenomination,
+    14,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/LimitPriceWarning/LimitPriceWarning.tsx]: I don't believe we need here, @jorbuedo ?
```suggestion
```

Comment (stackchain) [reply apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/LimitPriceWarning/LimitPriceWarning.tsx]: ~ow yeah, defined to 14, @banklesss please add as a constant on both prices, later we add to the module itself.~

Comment (stackchain) [reply apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/LimitPriceWarning/LimitPriceWarning.tsx]: aa u already did nvm

Comment (jorbuedo) [reply apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/LimitPriceWarning/LimitPriceWarning.tsx]: That's the number of decimals we show, probably not needed for the warning.

**Pull request merged**